pocs/utils/images/focus.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+
+
+def focus_metric(data, merit_function='vollath_F4', **kwargs):
```

Comment:
This seems to have hit the YAGNI level, as there is only a single merit function (which seems like it should be called metric_function).
Do you plan to add Vollath's F5 or other such metric?

Reply:
Pinging @AnthonyHorton on this one. I think the plan was to add more merit functions originally, not sure if that is still the plan.

Reply:
But don't think it should be changed as part of this PR regardless.

Reply:
Agreed, I'm just raising the issue now as this is the first time I've reviewed this code.

Reply:
@wtgee is correct. I expected to have to experiment with several different merit functions, and possibly even use more than one depending on the situation (e.g. one for coarse focus and a different one for fine focus). As it turned out the first thing I tried (Vollath's F4) seemed to work tolerably well, at least for fine focusing. It's not great though, and I do still expect to try some alternatives. https://github.com/AstroHuntsman/HuntsmanPOCSLegacy/issues/30

Good point on the nomenclature, though. Either metric and merit function would be appropriate in this context, but should settle on one of them instead of using both.